ports/azure-security-keyvault-keys-cpp/portfile.cmake

```
@@ -0,0 +1,19 @@
+vcpkg_from_github(
+    OUT_SOURCE_PATH SOURCE_PATH
+    REPO Azure/azure-sdk-for-cpp
+    REF azure-security-keyvault-keys_4.0.0-beta.1
+    SHA512 fb5158320c859f27c97c86908fd4c3f7ceec2ec8b0ca3c213a075992b62dfdefa3a560173bc7034153c273c30abd7fd0a84f98842810de2f78412549e5d34ee4

```

```suggestion
    SHA512 fb5158320c859f27c97c86908fd4c3f7ceec2ec8b0ca3c213a075992b62dfdefa3a560173bc7034153c273c30abd7fd0a84f98842810de2f78412549e5d34ee4
    HEAD_REF master
```
Please see https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/vcpkg_from_github.md#notes

---

@NancyLi1013, we use `REF`, why do we also need `HEAD_REF master`? We tag our releases, we don't release from master.

---

So you only use `REF`, which is not related with branch. right? If yes, `HEAD_REF ` might not be required.

---

Yes - we tag releases in master branch and release from there: https://github.com/Azure/azure-sdk-for-cpp/tags

---

Sorry for my wrong understanding about this. If we add `HEAD_REF`, will enable support for `install --head`.
https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/vcpkg_from_github.md#head_ref

So this is an extra feature. Adding it or not are both good.